src/transformStateWithClones.js

```diff
@@ -7,7 +7,38 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const states = [state];
+
+  for (let i = 0; i < actions.length; i++) {
```

Comment:
```suggestion
  for (const action of actions) {
```
Better use `for...of` instead of default loops for better readability